src/app/services/snackbar.service.ts

```diff
@@ -63,6 +63,10 @@ export class SnackbarService implements OnDestroy {
     );
   }
 
+  snackbarIsOpen(): Boolean {
+    return Boolean(this.snackBar._openedSnackBarRef);
+  }
```

Comment:
Thanks for adding this! 

I don't think you need to convert it to 'Boolean' here - it's fine to just return a 'boolean'.